force-app/main/default/reports/NPSP_Outbound_Fund_Reports/Unscheduled_Disbursements_for_Awarded_Gr.report-meta.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Report xmlns="http://soap.sforce.com/2006/04/metadata">
+    <columns>
+        <field>CUST_NAME</field>
+    </columns>
+    <columns>
+        <field>Funding_Request__c.Awarded_Date__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>Funding_Request__c.Awarded_Amount__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>Funding_Request__c.Total_Remaining__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>Funding_Request__c.Total_Disbursed__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>Funding_Request__c.Unpaid_Disbursements__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>Funding_Request__c.Total_Cancelled__c</field>
+    </columns>
+    <description>List of funding requests with available amount for disbursement.</description>
```

Comment:
```suggestion
    <description>List of Disbursements that are overdue by 1-30 days, 31-60 days, and 61+ days.</description>
```

Reply:
@nicolersmith - This is a different report. We aren't grouping on this report. We show funding requests that have available money to be disbursed.